app/main.py

```diff
@@ -1,3 +1,54 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+import os
+
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+
+    parent_dir = os.path.join(
+        "C:/",
+        "self_learning",
+        "Python",
+        "py-shop-trip",
+        "app"
+    )
```

Comment:
No it is not good. Because it will not work on my computer, because you say your path.
You can do:
parent_dir = os.path.dirname(__file__)